src/components/CheckList/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import CheckMark from '../../img/check-icon.svg';
+
+const Wrapper = styled.div`
+	margin-top: 66px;
+	display: grid;
+	grid-template-columns: repeat(2, 1fr);
+	grid-template-rows: auto;
+	grid-column-gap: 80px;
+	grid-row-gap: 25px;
+	@media(max-width: 600px) {
+		margin-top: 55px;
+		grid-template-columns: repeat(1, 1fr);
+	}
+`
+const CheckItem = styled.div`
+	background: linear-gradient(270deg, rgba(69, 183, 210, 0) 0%, #226CC7 100%);
+	border-radius: 40px;
```

Comment:
Try to use outside-in CSS organisation. We try to follow these guidelines in all our projects, so it's easier for everyone to read CSS. Please resolve this in all scss

https://webdesign.tutsplus.com/articles/outside-in-ordering-css-properties-by-importance--cms-21685

Reply:
okay